index.html

```diff
@@ -1,13 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    
-    <title>To-Do-Manager</title>
-    <!-- Favicon icon link -->
-    <link rel="icon" href="./assets/images/to-do icon.png" type="image/x-icon">
```

Comment:
merge commit [b10c644](https://github.com/malivinayak/To-Do-Manager/pull/50/commits/b10c644729fef9f3a05b305624be98de87018720) removed the icon code